src/style.css

```diff
@@ -1 +1,135 @@
-/* add styles here */
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir';
+  font-weight: 300;
+  font-style: normal;
+  src: url('../src/fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir';
+  font-weight: 600;
+  font-style: normal;
+  src: url('../src/fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+form {
+  display: block;
+  unicode-bidi: isolate;
+}
+
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Avenir', sans-serif;
+  font-size: 16px;
+}
+
+label {
```

Comment:
Avoid using tag names for styling. Instead, use class selectors to apply styles to label elements.

Reply:
Same here.